### ISSUE TITLE
Deleting first char while editing messages

### ISSUE THREAD

**Issue by czalaviek** [mudita/MuditaOS#3448]
# 🐛 Bug report

## 📝 Description

Pressing del key [#], when cursor is located on 1'st position, removes 1'st right char.

## 👍 Expected behavior

Del key should do nothing when cursor is located on the start position.

## 👎 Current behavior

Pressing del key [#], when cursor is located on 1'st position, removes 1'st right char.

## 🔬 Minimal reproduction

Situation is so simple and it doesn't need long description :)

## 🔍 Provide logs

<pre><code>

</code></pre>

## 🌍 Your operating system and MuditaOS version
1.1.5

## 🤔 Anything else relevant?

**Comment by czalaviek** [comment]
IT looks that error od harder, because the same situation is any textbox created from the same stereotype, ex. fields in contacts.

**Comment by pholat** [comment]
This is dependent on how the UI text widget is used in particular cases. Generally speaking, behavior when we are with cursor prior to any other character, we press delete, we remove character after (not prior as there is none) was left as a feature in code as it was proven handy - it's hard to press `backspace` by accident on the phone keyboard too. Therefore, it didn't feel as it would bother anyone.
Could you please point out specific places where it's bothering you and why?

**Comment by czalaviek** [comment]
Cases:
1) Editing new message - I wrote "Horse", but I wanted to write "Morse". To correct the word, I'm moving cursor to position between "H" and "o", next move is hitting "#" as Backspace. Unfortunately I pressed it twice. Result is deleted char "H" but "o" too. It's unwanted behaviour, because needs to write two chars, now.
The same behaviour is while editing Name, Surname etc. in contacts.

Pure's keyboard isn't ASCII-keyboard device so every possibility to avoid unneeded keystrokes is good decision.
